compiler/luci-interpreter/src/KernelBuilder.cpp

```diff
@@ -180,6 +181,16 @@ std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleFullyConnected *n
   return std::make_unique<kernels::FullyConnected>(input, filter, bias, output, params);
 }
 
+std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleL2Normalize *node)
+{
+  assert(node->arity() == 1);
+
+  const Tensor *input = getInputTensor(node->x());
+  Tensor *output = getOutputTensor(node);
+
+  return std::make_unique<kernels::L2Normalize>(input, output);
```

Comment:
`CircleL2Normalize` is inherited from `LuciNodeMixin<LuciNodeTrait::FusedActFunc>`, so we have `fusedActivationFunction` attribute for this operator.
Need to support it in kernels, or add assert/throw, so we will not miss.

Reply:
Note that TFLite does not support activations other than kNone:
https://github.com/tensorflow/tensorflow/blob/e5bf8de410005de06a7ff5393fafdf832ef1d4ad/tensorflow/lite/kernels/l2norm.cc#L64-L66

Reply:
Sorry for missing that. I know only kNone is supported, but forget to write param about this. I will make update commit for this. sorry for make confused.